postgrest.cabal

```
@@ -87,7 +87,7 @@ library
                     , hasql-dynamic-statements  == 0.3.1
                     , hasql-notifications       >= 0.1 && < 0.2
                     , hasql-pool                >= 0.5 && < 0.6
-                    , hasql-transaction         >= 0.7.2 && < 1.1
+                    , hasql-transaction         >= 1.0.1
```

Needs an upper bound:
```suggestion
                    , hasql-transaction         >= 1.0.1 && < 1.1
```